fcitx.el

```diff
@@ -505,14 +505,18 @@ Default value is nil.")
 (defvar fcitx--aggressive-p nil
   "Whether we should disable fcitx whenever we're in the minibuffer.")
 
+(defvar fcitx-remote-command "fcixt-remote"
```

Comment:
I think there is a wrong grammar here. Must be "fcitx-remote".

Reply:
Good catch! Fixed now.